build/commands/lib/config.js

```diff
@@ -848,6 +848,12 @@ Config.prototype.update = function (options) {
   if (options.target) {
     this.buildTarget = options.target
   }
+  if (options.use_pgo) {
+    this.chromePgoPhase = 2 // Download and use chromium PGO profiles
+  } else {
+    // Temporary enable by default.
+    this.chromePgoPhase = 2 // Do not use PGO
```

Comment:
you have the same value for both options

Reply:
yeah. It's just for testing on CI, see the second commit. I will revert this.

Reply:
Done